packages/autorest.python/samples/specification/management/generated/azure/mgmt/sample/aio/operations/_http_success_operations.py

```diff
@@ -41,10 +44,10 @@ class HttpSuccessOperations:
 
     def __init__(self, *args, **kwargs) -> None:
         input_args = list(args)
-        self._client = input_args.pop(0) if input_args else kwargs.pop("client")
-        self._config = input_args.pop(0) if input_args else kwargs.pop("config")
-        self._serialize = input_args.pop(0) if input_args else kwargs.pop("serializer")
-        self._deserialize = input_args.pop(0) if input_args else kwargs.pop("deserializer")
+        self._client: AsyncARMPipelineClient = input_args.pop(0) if input_args else kwargs.pop("client")
```

Comment:
After https://github.com/Azure/azure-sdk-for-python/pull/31584 merged, we need to upgrade azure.mgmt.core version and change this line to `AsyncARMPipelineClient[HttpRequst, AsyncHttpResponse]`